nifi-registry/nifi-registry-core/nifi-registry-jetty/src/main/java/org/apache/nifi/registry/jetty/JettyServer.java

```diff
@@ -147,9 +151,39 @@ private File getDocsDir(final String docsDirectory) {
     }
 
     private void configureConnectors() {
-        final ServerConnectorFactory serverConnectorFactory = new ApplicationServerConnectorFactory(server, properties);
-        final ServerConnector serverConnector = serverConnectorFactory.getServerConnector();
-        server.addConnector(serverConnector);
+        try {
```

Comment:
This outer try-catch block is unnecessary because `configureConnectors()` is already called from a try-catch with the same handling.

Reply:
Making the change